terminology.rst

```diff
@@ -116,6 +116,10 @@ Raiden Terminology
    Locked Transfer message
        An offchain Raiden message that reserves an amount of tokens for a specific :term:`Payment`. See :ref:`locked-transfer-message` for details.
 
+   Refund Transfer
+   Refund Transfer message
+       An offchain Raiden message for a :term:`Payment` seeking a rerouting. When a receiver of a :term:`Locked Transfer` message gives up reaching the target, she returns a Refund Transfer message. A Refund Transfer message locks an amount of tokens in the same way as a :term:`Locked Transfer` message does.
```

Comment:
/s/she/they

Reply:
Also perhaps add "but in the opposite direction" in the end.

Reply:
I changed `she` into `they`.

I didn't add "the opposite direction" because, to me, "but in the opposite direction" sounded confusing.  (The sender of the Refund message can lock amounts coming from the receiver of the message?)

Reply:
What I meant is that for a refund transfer the receiver of the transfer sends a locked transfer with the same amounts back to the payer.

Reply:
I see.  That explains the name "refund" well.  Let me figure out something.

Reply:
Done.  I think this works: https://github.com/raiden-network/spec/pull/174/files#diff-a84d6f345909c58c851843f12aa76f5fR123.